pkg/pool-stable-phantom/contracts/StablePhantomPool.sol

```diff
@@ -463,10 +470,14 @@ contract StablePhantomPool is StablePool {
         cache = PriceRateCache.encode(rate, duration);
     }
 
-    /**
-     * @dev Overrides BasePool's minimum BPT amount since all BPT is minted to the Vault during initialization.
-     */
-    function _getMinimumBpt() internal pure override returns (uint256) {
-        return _MINIMUM_BPT;
+    function _skipBptIndex(uint256 index) internal view returns (uint256) {
+        return index < _bptIndex ? index : index - 1;
+    }
+
+    function _dropBptItem(uint256[] memory _balances) internal view returns (uint256[] memory balances) {
+        balances = new uint256[](_balances.length - 1);
+        for (uint256 i = 0; i < balances.length; i++) {
+            balances[i] = _balances[_skipBptIndex(i)];
```

Comment:
For the `_dropBptItem ` it was ok to have `index + 1` :) 

Maybe because it is used only here you can do it inline:
`balances[i] = _balances[i < _bptIndex ? i : i + 1];`

Reply:
I forgot to push that last part, see my last commit please :)